### ISSUE TITLE
cannot install - stack Error: `make` failed with exit code: 2

### ISSUE THREAD

**Issue by cleechtech** [asbjornenge/thinking-in-react#15]
I have tried installing this package with node 4.0.0 and 4.2.1 to no avail. Also tried installing it with and without sudo, clearing the cache and a few other commands. I installed `learnyoureact` globally with no issues. I do not know why I cannot install this package. I am running OSX.

```
make: *** [Release/obj.target/contextify/src/contextify.o] Error 1
gyp ERR! build error 
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/usr/local/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:270:23)
gyp ERR! stack     at emitTwo (events.js:87:13)
gyp ERR! stack     at ChildProcess.emit (events.js:172:7)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:200:12)
gyp ERR! System Darwin 15.0.0
gyp ERR! command "/usr/local/bin/node" "/usr/local/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /usr/local/lib/node_modules/thinking-in-react/node_modules/contextify
gyp ERR! node -v v4.0.0
gyp ERR! node-gyp -v v3.0.3
gyp ERR! not ok 
npm ERR! Darwin 15.0.0
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "install" "-g" "thinking-in-react"
npm ERR! node v4.0.0
npm ERR! npm  v3.3.6
npm ERR! code ELIFECYCLE

npm ERR! contextify@0.1.14 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the contextify@0.1.14 install script 'node-gyp rebuild'.
npm ERR! This is most likely a problem with the contextify package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node-gyp rebuild
npm ERR! You can get their info via:
npm ERR!     npm owner ls contextify
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/cleech/npm-debug.log
```

**Comment by asbjornenge** [comment]
I've seen this before when upgrading to node 4. It seems it's the `jsdom` package that has `contextify` as a dependency. There are newer versions on jsdom, I'll look into upgrading. Report back shortly.

**Comment by asbjornenge** [comment]
Ok, just published a new version v3.0.0 to npm. Should work with node 4+ :smile: Let me know if you have issues!

**Comment by cleechtech** [comment]
Works now. Thank you
